x-pack/legacy/plugins/maps/public/layers/styles/vector/properties/dynamic_color_property.test.js

```diff
@@ -260,56 +260,77 @@ describe('get mapbox color expression', () => {
       test('should return null when color ramp is not provided', async () => {
         const dynamicStyleOptions = {
           type: COLOR_MAP_TYPE.ORDINAL,
-          field: {
-            name: 'myField',
-          },
         };
         const colorProperty = makeProperty(dynamicStyleOptions);
         expect(colorProperty._getMbColor()).toBeNull();
       });
 
-      test('should return mapbox expression for color ramp', async () => {
-        const dynamicStyleOptions = {
-          type: COLOR_MAP_TYPE.ORDINAL,
-          field: {
-            name: 'myField',
-          },
+      test('Should set the correct mapbox style', async () => {
+        const colorStyle = makeProperty({
           color: 'Blues',
+          type: undefined,
+        });
+
+        const calls = [];
+        const mockMbMap = {
+          setPaintProperty(...args) {
+            calls.push(args);
+          },
         };
-        const colorProperty = makeProperty(dynamicStyleOptions);
-        expect(colorProperty._getMbColor()).toEqual([
-          'interpolate',
-          ['linear'],
-          ['coalesce', ['feature-state', '__kbn__dynamic__myField__lineColor'], -1],
-          -1,
-          'rgba(0,0,0,0)',
-          0,
-          '#f7faff',
-          0.125,
-          '#ddeaf7',
-          0.25,
-          '#c5daee',
-          0.375,
-          '#9dc9e0',
-          0.5,
-          '#6aadd5',
-          0.625,
-          '#4191c5',
-          0.75,
-          '#2070b4',
-          0.875,
-          '#072f6b',
+
+        colorStyle.syncCircleColorWithMb('foobarid', mockMbMap, 1);
```

Comment:
The entire describe block that this test is in is specifically for `_getMbColor`. This should either be a new test block or switched back to `_getMbColor`